pyQuARC/code/custom_validator.py

```diff
@@ -161,7 +161,7 @@ def collection_progress_consistency_check(collection_state, ends_at_present_flag
         collection_state = collection_state.upper()
         ending_date_time_exists = bool(ending_date_time)
         ends_at_present_flag_exists = bool(ends_at_present_flag)
-        ends_at_present_flag = ends_at_present_flag_exists.lower() if ends_at_present_flag_exists else None
+        ends_at_present_flag = str(ends_at_present_flag).lower() if ends_at_present_flag_exists else None
```

Comment:
`str()` because apparently you can sometimes receive bool too. also, missed an error from last PR (use of `ends_at_present_flag_exists` instead of `ends_at_present_flag`.